pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -292,6 +293,20 @@ func buildHTTPVirtualServices(
 			vs.Route = route
 		}
 
+		if r.Timeouts != nil {
+			if r.Timeouts.Request != nil {
```

Comment:
Need to update this now that GEP-1742 timeout value 0 -> disable.
```suggestion
			if r.Timeouts.Request != nil && r.Timeouts.Request.Duration != 0 {
```